lib/Event/EventOptionsTrait.php

```diff
@@ -19,14 +21,14 @@
     /**
      * This array is used as key value storage for the options.
      *
-     * @var array
+     * @var OptionsResolver
```

Comment:
You can, exactly the same way it is done in this class' `getOption` method. The `OptionsResolver` implements `\ArrayAccess`.